docs/buildah-manifest-push.1.md

```diff
@@ -18,6 +18,10 @@ The list image's ID and the digest of the image's manifest.
 
 ## OPTIONS
 
+**--add-compression** *compression*
+
+If specified makes sure while pushing that requested compression variant exists for each platform in the manifest list.
```

Comment:
This might not be the ideal place, but _somewhere_ we should document the accepted algorithm names, so that users know what values to use.

Reply:
Probably in `cli` flags description ? WDYT ?

Reply:
I’m not sure what ”`cli` flags description” refers to. Is that some man page, or just an internal Go documentation? The commend was targeting user-visible documentation, like man pages.

Short-term, we don’t _need_ to overcomplicate things — just list the accepted values (`gzip`, `zstd`, `zstd:chunked` (and hypothetically `Xz`, but I think that doesn’t work because we can’t represent it in manifests) here in this option description.

---

Currently `pkg/compression` does not provide an API to list accepted names. That might be worth adding.

Reply:
@mtrmac I mean the flag description which shows when we do `buildah manifest push --help` but I think documentation should also mention it. So added here instead.